libraries/launcher/net/minecraft/Launcher.java

```diff
@@ -84,7 +84,6 @@ public Launcher(Applet applet) {
     }
 
     public Launcher(Applet applet, URL documentBase) {
-        super();
```

Comment:
We should explicitly have `super();` in the constructor. It's good practice, as otherwise there's just an implied super call. (Plus, my IDE complains when it's not there)

Reply:
In the majority of code styles, this isn't an issue. I don't see any reason to add something that doesn't change anything, since it's already fairly apparent that the class extends something else. In my experience, IDEs don't warn about it by default. However, I'm not really that bothered either way.